src/modal/alert-modal.component.ts

```diff
@@ -118,10 +118,11 @@ export class AlertModal extends BaseModal implements AfterViewInit {
 		this.closeModal();
 	}
 
-	dismissModal() {
+	dismissModal(trigger) {
```

Comment:
At the moment the trigger parameter doesn't appear to be doing anything. I think onClose() needs to take in trigger as an argument?